flink-core/src/main/java/org/apache/flink/api/common/state/AggregatingState.java

```diff
@@ -33,7 +33,7 @@
  * <p>The state is accessed and modified by user functions, and checkpointed consistently
  * by the system as part of the distributed snapshots.
  * 
- * <p>The state is only accessible by functions applied on a KeyedDataStream. The key is
+ * <p>The state is only accessible by functions applied on a KeyedStream. The key is
```

Comment:
Could you also change these to be actual Javadoc links?
i.e. `{@link KeyedStream}`.